tools/devtool

```diff
@@ -737,6 +739,24 @@ cmd_build() {
     return $ret
 }
 
+cmd_distclean() {
+    # List of folders to remove.
+    dirs=("build" "test_results")
+
+    for dir in "${dirs[@]}"; do
+        if [ -d "$dir" ]; then
+            say "Removing $dir"
+            rm -rf "$dir"
+        fi
+    done
+
+    # Remove devctr if it exists
+    if [ $(docker images -q "$DEVCTR_IMAGE" | wc -l) -eq "1" ]; then
+        say "Removing $DEVCTR_IMAGE"
+        docker rmi "$DEVCTR_IMAGE"
```

Comment:
Should we force remove here ? Sometimes the container continues to run, and we'll not be able to remove the image otherwise.

Reply:
Sure, I'll add the `-f` flag.